botocore/handlers.py

```diff
@@ -749,5 +749,7 @@ def convert_body_to_file_like_object(params, **kwargs):
           'PutBucketLifecycle', 'PutBucketLogging', 'PutBucketNotification',
           'PutBucketPolicy', 'PutBucketReplication', 'PutBucketRequestPayment',
           'PutBucketTagging', 'PutBucketVersioning', 'PutBucketWebsite',
-          'PutObjectAcl']).hide_param)
+          'PutObjectAcl']).hide_param),
+    ('docs.*.iam.CreateUser.complete-section',
```

Comment:
I think this needs to get undocumented in a bit more locations like ``get_group``, ``create_virtual_mfa_device`` and ``list_virtual_mfa_devices`` and possibly some other locations. I wonder if the list should be exclusive as opposed to inclusive such that all instances where the User shape and the operation is not GetUser or ListUser then apply the handler? It may be easier then auditing everything, but may lead to unintended consequences such as if User is used say in the input to an operation or something.